spec/models/video_spec.rb

```diff
@@ -2,98 +2,56 @@
+  it { is_expected.to validate_presence_of :description }
+  it { is_expected.to have_many :reviews }
+
+  describe ".search_by_title" do
+    let!(:videos) do
+      videos = 5.times.map { |i| Fabricate(:video, title: "Lost #{i+1}") }
+    end
+    subject { lambda {|title| Video.search_by_title(title)} }
 
-  describe "search_by_title" do
     it "returns an empty array if there are no matches" do
-      Video.create!(title: "Lost", description: "So lost")
-      Video.create!(title: "Lost 2", description: "Soooo lost")
-      Video.create!(title: "Lost 3", description: "So very lost")
-      Video.create!(title: "Family Guy", description: "Hahaha")
-      Video.create!(title: "Futurama", description: "Woooo")
-
-
-      expect(Video.search_by_title("no such title")).to eq([])
+      expect(subject.call("no such title")).to eq([])
     end
 
     it "finds one video if there is only one match in videos" do
-      Video.create!(title: "Lost", description: "So lost")
-      Video.create!(title: "Lost 2", description: "Soooo lost")
-      Video.create!(title: "Lost 3", description: "So very lost")
-      Video.create!(title: "Family Guy", description: "Hahaha")
-      Video.create!(title: "Futurama", description: "Woooo")
-
-      search_result = Video.search_by_title("Family Guy")
-
-      expect(search_result.size).to eq(1)
+      expect(subject.call("Lost 5").size).to eq(1)
     end
 
     it "finds video that has a partially matched title" do
-      Video.create!(title: "Lost", description: "So lost")
-      Video.create!(title: "Lost 2", description: "Soooo lost")
-      Video.create!(title: "Lost 3", description: "So very lost")
-      Video.create!(title: "Family Guy", description: "Hahaha")
-      Video.create!(title: "Futurama", description: "Woooo")
-
-      search_result = Video.search_by_title("Family")
-
-      expect(search_result.size).to eq(1)
+      expect(subject.call("ost 5").size).to eq(1)
     end
 
     it "finds all videos that matches the title" do
-      Video.create!(title: "Lost", description: "So lost")
-      Video.create!(title: "Lost 2", description: "Soooo lost")
-      Video.create!(title: "Lost 3", description: "So very lost")
-      Video.create!(title: "Family Guy", description: "Hahaha")
-      Video.create!(title: "Futurama", description: "Woooo")
-
-      search_result = Video.search_by_title("Lost")
-
-      expect(search_result.size).to eq(3)
+      expect(subject.call("Lost").size).to eq(5)
     end
 
     it "ignores letter cases" do
-      Video.create!(title: "Lost", description: "So lost")
-      Video.create!(title: "Lost 2", description: "Soooo lost")
-      Video.create!(title: "Lost 3", description: "So very lost")
-      Video.create!(title: "Family Guy", description: "Hahaha")
-      Video.create!(title: "Futurama", description: "Woooo")
-
-      search_result_lost = Video.search_by_title("lOst")
-
-      expect(search_result_lost.size).to eq(3)
+      expect(subject.call("lOst").size).to eq(5)
```

Comment:
I personally do not prefer using `subject` it abstracts a lot from the spec to my taste, so I would have to end up checking what subject is all the time.